src/nav/mobile-monitoring.yml

```
@@ -7,36 +7,30 @@ pages:
     pages:
       - title: Android
         pages:
-          - title: Get started
+          - title: Install and get started
             pages:
-              - title: Introduction to Android monitoring
+              - title: Install and begin Android monitoring
                 path: /docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android
               - title: Compatibility and requirements
                 path: /docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements
+              - title: Installation for Instant Apps
+                path: /docs/mobile-monitoring/new-relic-mobile-android/get-started/install-new-relic-plugin-android-instant-apps
               - title: Android distributed tracing
                 path: /docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-and-dt
               - title: Android release notes
                 path: /docs/mobile-monitoring/new-relic-mobile-android/get-started/android-release-notes
-          - title: Install and configure
+          - title: Configuration
             pages:
-              - title: Gradle and Android Studio installation
-                path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio
+              - title: Configure Android apps with Gradle and Android Studio
+                path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio
               - title: Configure the New Relic Gradle plugin
                 path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-new-relic-gradle-plugin
               - title: Configure ProGuard and DexGuard
                 path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps
               - title: Android crash reporting
                 path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting
-              - title: Installation for Instant Apps
-                path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-new-relic-plugin-android-instant-apps
-              - title: Update the Android SDK
-                path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/upgrade-new-relic-mobiles-android-sdk
               - title: Configure app launch time metrics
                 path: /docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps
-          - title: Legacy
-            pages:
-              - title: Legacy Android 2.2 support
-                path: /docs/mobile-monitoring/new-relic-mobile-android/legacy/install-android-apps-android-22-support
           - title: Troubleshoot
```

```suggestion
          - title: Troubleshooting
```

To match "configuration" as a gerund